ui/app/templates/jobs/run/templates/new.hbs

```diff
@@ -0,0 +1,34 @@
+<section class="section">
+    <form class="new-job-template" autocomplete="off">
+        <div class={{if this.namespaceOptions "path-namespace"}}>
+            <label>
+            <span>
+                Template name
+            </span>
+            <Input
+                @type="text"
+                @value={{mut this.templateName}}
+                placeholder="your-template-name-here"
+                class="input path-input"
+                {{autofocus}}
+                data-test-template-name
+            />
+            </label>
+            <SingleSelectDropdown
+                data-test-namespace-facet
+                @label="Namespace"
+                @options={{this.namespaceOptions}}
+                @selection={{this.templateNamespace}}
+                @onSelect={{fn (mut this.templateNamespace)}}
+            />
+        </div>
+        <VariableForm::JobTemplateEditor
+            @keyValues={{this.model.keyValues}}
+            @updateKeyValue={{this.updateKeyValue}}
+        />
+        <footer>
```

Comment:
Margin between editor and footer please 
![image](https://user-images.githubusercontent.com/713991/211636740-4a4eaebe-2717-4446-98ae-cf7007f304f9.png)